q2_fondue/tests/test_efetch.py

```diff
@@ -264,6 +264,9 @@ def test_efetch_to_df_duplicated_columns(self):
             'STUDY1': MagicMock(generate_meta=MagicMock(return_value=dfs[0])),
             'STUDY2': MagicMock(generate_meta=MagicMock(return_value=dfs[1]))
         }
+        self.efetch_result_single.runs = {
```

Comment:
Is there anything that speaks against adding an additional test where the fetched metadata contains more IDs than the requested ones (as seems to be the case for when this bug occurs)?

Reply:
Absolutely not! I got overexcited and kinda forgot 😅 Will add 👍